src/App.tsx

```diff
@@ -1,24 +1,312 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Filter } from './components/Filter';
+import { ShowTodos } from './types/ShowTodos';
+import { ShowError } from './types/ShowErrors';
 
-const USER_ID = 0;
+const USER_ID = 10565;
+
+const todosFromServer = client
+  .get<Todo[]>(`/todos?userId=${USER_ID}`);
+const createTodoOnServer = (title: string) => client
+  .post<Todo>('/todos', {
+  title,
+  userId: USER_ID,
+  completed: false,
+});
+const deleteTodoOnServer = (todoId: number) => client
+  .delete(`/todos/${todoId}`);
+const toggleTodoOnServer = (todoId: number, completed: boolean) => client
+  .patch<Todo>(`/todos/${todoId}`, { completed: !completed });
+const updateTodoOnServer = (todoId: number, title: string) => client
+  .patch<Todo>(`/todos/${todoId}`, { title });
```

Comment:
it should be in `api/todos.ts` file
never create handlers for api in main App

Reply:
Done